jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/handler/ResourceHandlerTest.java

```diff
@@ -1480,6 +1480,7 @@ public void testCachingPrecompressedFilesCachedEtagged() throws Exception
                     \r
                     """.formatted(eTag1)));
             assertThat(response3.getStatus(), is(HttpStatus.NOT_MODIFIED_304));
+            assertThat(response3.getField(ETAG), notNullValue());
```

Comment:
Shouldn't `Content-Length` and `Last-Modified` be asserted too?

Reply:
The RFC says ETAG or Last-Modified and stresses we should minimize bytes.
It does not say we SHOULD/MUST send content-length, but says it MAY be sent.